devices.js

```diff
@@ -9571,6 +9571,17 @@ const devices = [
             await reporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['ICZB-KPD12'],
+        model: 'ICZB-KPD12',
+        vendor: 'iCasa',
+        description: 'Zigbee 3.0 Keypad Pulse 2',
+        meta: {battery: {dontDividePercentage: true}},
+        fromZigbee: [fz.command_on, fz.legacy.genOnOff_cmdOn, fz.command_off, fz.legacy.genOnOff_cmdOff, fz.battery,
```

Comment:
Please don't use the `fz.legacy` here.
- `fz.legacy.genOnOff_cmdOn`, `fz.legacy.genOnOff_cmdOff` can be removed
- `fz.legacy.cmd_move_with_onoff` -> `fz.command_move`
- `fz.legacy.cmd_stop_with_onoff` -> `fz.command_stop`

Reply:
Hm, ok. But for sake of consistency, shouldn't those also be removed/replaced for the Pulse 4S and 8S then? Those products are very similar and this is where I copied these conversions from.

Reply:
No because that will create a breaking change for existing users.